File: hw/Pérez/U2/HW18ZooJavaScript/model/Eagle.js
class Eagle extends Bird {
    constructor(id, description, bornOnDate, gender, cage, numberOfBones, vetebrate) {
        super(id, description, bornOnDate, gender, cage, numberOfBones, vetebrate);
    }

    feed(food) {
        console.log("Feeding the eagle with mice");
    }
}

File: ws/segarra/u2/WS21Inheretance/WS21Inheritance/src/main/java/ec/edu/espe/WS21Inheritance/model/Eagle.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.WS21Inheritance.model;

import java.util.Date;

/**
 *
 * @author Eduardo Segarra, TheJavaBandits, DCCO-ESPE
 */
public class Eagle extends Bird{

    public Eagle(int id, String description, Date bornOnDate, char gender, Cage cage, int numberOfBones, boolean vetebrate) {
        super(id, description, bornOnDate, gender, cage, numberOfBones, vetebrate);
    }

    @Override
    public void feed(int food) {
        System.out.println("feeding the eagle with mice");
    }
    
}
